src/control/cmd/daos_agent/start.go

```diff
@@ -65,7 +65,6 @@ func (cmd *startCmd) Execute(_ []string) error {
 		cmd.log.Debugf("GetAttachInfo agent caching has been disabled\n")
 	}
 
-	netdetect.SetLogger(cmd.log)
```

Comment:
This line made it into start.go due to a bad merge.  It was supposed to be removed.  It has no impact on the defect.